forge-gui/res/cardsfolder/upcoming/getaway_car.txt

```diff
@@ -0,0 +1,17 @@
+Name:Getaway Car
+ManaCost:3
+Types:Artifact Vehicle
+PT:4/3
+K:Haste
+T:Mode$ BecomesCrewed | ValidVehicle$ Card.Self | Execute$ RememberCrew | Static$ True
+SVar:RememberCrew:DB$ PumpAll | ValidCards$ Creature.TriggeredCrew | RememberAllPumped$ True
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Any | ValidCard$ Card.Self | Execute$ DBCleanup | Static$ True
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ Player | TriggerZones$ Battlefield | Execute$ DBCleanup | Static$ True
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigReturn | TriggerDescription$ Whenever CARDNAME attacks or blocks, return up to one target creature that crewed it this turn to its owner’s hand.
+T:Mode$ Blocks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Secondary$ True | Execute$ TrigReturn | TriggerDescription$ Whenever CARDNAME attacks or blocks, return up to one target creature that crewed it this turn to its owner’s hand.
+SVar:TrigReturn:DB$ ChangeZone | ValidTgts$ Creature.IsRemembered | TargetMin$ 0 | TargetMax$ 1 | Origin$ Battlefield | Destination$ Hand | TgtPrompt$ Select up to one target creature

```

Comment:
Can you make this TgtPrompt more specific?

I'd like to do this with `ValidTgts$ Creature.crewedThisTurn` and eliminate the Static trigger, but this should be OK for now